WORKING_GROUPS.md

```diff
@@ -270,6 +271,29 @@ Its responsibilities are:
 + Defining and adding common structures to the dumps generated
   in order to support tools that want to introspect those dumps
 
+### [Inclusivity](https://github.com/nodejs/inclusivity)
+
+The Inclusivity Working Group seeks to increase inclusivity and diversity
+for the Node.js project:
+  - *Increasing inclusivity* means making the Node.js project a safe and
+    friendly place for people from diverse backgrounds.
+  - *Increasing diversity* means actively onboarding people from diverse
+    backgrounds to the Node.js project and maintaining their participation.
+
+Its responsibilites are:
+
++ Foster a welcoming environment that ensures participants are valued and can
+feel confident contributing or joining discussions, regardless of any aspect of 
+their identity.
++ Proactively seek and propose concrete steps the project can take to increase
+inclusivity.
++ Serve as a resource for the development and enforcement of workflows that 
```

Comment:
I still have issues with the use of the word `enforcement` here. I recommend splitting this into two pieces and rewording to, 
```
+ Serve as a resource for the development of workflows that protect community members and projects from harassment and abuse.
+ Work within the project and the community at large to ensure effective implementation of those workflows.
```
The issue with using `enforcement` is that it implies the ability to take punitive action and generally carries negative connotation.